haystack/pipelines/config.py

```diff
@@ -123,6 +124,8 @@ def validate_config_strings(pipeline_config: Any, is_value: bool = False):
                         json.loads(value)
                     except json.decoder.JSONDecodeError as e:
                         raise PipelineConfigError(f"'{pipeline_config}' does not contain valid JSON.")
+                elif key in SKIP_VALIDATION_KEYS:
+                    continue
```

Comment:
What do you mean by annoying people? We can't load the template text without this skip.

Reply:
Exactly: I mean that we should rethink/remove the regex validation. At least evaluate properly what value it really brings, because it's not clear anymore.